src/parser/tests/lexer.rs

```diff
@@ -116,9 +116,9 @@ fn lex_string_concat() {
     let inputs = vec![
         "'hello' ~ `hey`",
         "'hello' ~ 1",
-        "'hello' ~ 3.14",
+        "'hello' ~ 3.18",
         "1 ~ 'hello'",
-        "3.14 ~ 'hello'",
+        "3.18 ~ 'hello'",
```

Comment:
why those changes?

Reply:
Clippy assumes 3.14 is a rounded pi and throws errors. 3.18 is a simple replacement.